src/OnboardingSPA/components/GridView/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Common Grid View Component
+ * Shows the components in a grid view
+ * Supports column size 2 - 5
+ *
+ * @return GridView of children
+ */
+
+const GridView = ( { size, colGap = 0, children } ) => {
```

Comment:
Let's add a custom `className` prop as well? We can concatenate with `grid-basic grid-col-${ size }` using `classnames`. I'm thinking this will give consumers better control over the component. Also, should we name it `Grid` instead of `GridView`?

Reply:
Sure, Thanks